examples/index.html

```diff
@@ -57,6 +57,10 @@
       .custom-hint-class {
         box-shadow: inset 0 0 0 5px #44952e;
       }
+
+      .custom-menu-class {
+        box-shadow: 0 3px 0 0 #fd0, 3px 3px 0 0 #fd0, -3px 3px 0 0 #fd0;
```

Comment:
Bit wordy, but extends the yellow focus border to wrap around the menu, not just the input, for a nice visual change.